src/utils/viewer/viewer.cpp

```diff
@@ -105,6 +113,12 @@ class Viewer : public Magnum::Platform::Application {
   bool drawObjectBBs = false;
 
   Magnum::Timeline timeline_;
+
+  // performance counter
+  bool displayPerformanceInfo = true;
+  uint64_t totalFrames_ = 0;
+  float timeDuration_ = 0.0f;
```

Comment:
What is this for? Never used, as far as I could see.

Reply:
Oh, will remove. 
This is because I read your previous message, and tried to introduce a variable to sum up the time elapsed since start() was called. But it is not needed.